azure-batch/HISTORY.rst

```diff
@@ -3,6 +3,24 @@
 Release History
 ===============
 
+5.0.0 (2018-08-24)
+++++++++++++++++++
+
+- Using REST API version 2018-08-01.7.0.
+  - Added `node_agent_info` in ComputeNode to return the node agent information
+  - **Breaking** Removed the `validation_status` property from `TaskCounts`.
+  - **Breaking** The default caching type for DataDisk and OSDisk is now `read_write` instead of `none`.
+- `BatchServiceClient` can be used as a context manager to keep the underlying HTTP session open for performance.
+- **Breaking** Model signatures are now using only keywords-arguments syntax. Every positional arguments are required to be rewritten as keywords-arguments.
```

Comment:
"Every positional arguments..." -> "Each positional argument must be rewritten as a keyword argument"